interfaces/cython/cantera/test/test_onedim.py

```diff
@@ -300,6 +300,16 @@ def test_soret_flag(self):
         self.sim.transport_model = 'Multi'
         self.sim.soret_enabled = True
 
+    def test_set_mix_with_soret(self):
+        # Test that setting mixture averaged transport with
+        # Soret diffusion enabled raises an error.
+
+        self.create_sim(101325, 300, 'H2:1.0, O2:1.0')
+        self.sim.transport_model = 'Multi'
+        self.sim.soret_enabled = True
+        with self.assertRaises(ct.CanteraError):
+            self.sim.transport_model = 'Mix'
```

Comment:
Right, so I guess my suggestion about how this should work breaks the ability to write fast tests...

Reply:
Unfortunately, GitHub hid my previous comment: https://github.com/Cantera/cantera/pull/511#discussion_r172738327 and I didn't see these down here until just now... Can you clarify that other comment?